### ISSUE TITLE
Turn off the group difficulty changes to prevent reveals of players underground e.t.c.

### ISSUE THREAD

**Issue by Archer-Dante** [SilentChaos512/ScalingHealth#225]
## Version

1.12.2 at least

## What Should Be Added or Changed?

For now there is no way to NOT display to players difficulty changes based on average of group and keep to show only player's personal difficulty, that causes some unpleasant situations.
First of all, it's easily reveals players position under ground when somebody just walking or flying around surfuce.
Turning off the whole difficulty display is not a solution since players still want to be able to track difficulty at any time.

## Why Should This Be Done?

Players can use their hidden or underground homes without being revealed just by this mod.

**Comment by Cyborgmas** [comment]
I am not doing dev work for 1.12 sadly, but in 1.14 I will add a button only config, that way you'll have the difficulty displayed only on that button press

**Comment by Archer-Dante** [comment]
well, button mean still no way to hide it completely?

**Comment by Cyborgmas** [comment]
Ah I misunderstood your question then. You mean that if you are at x difficulty currently and you go near a player, then that difficulty will not change to 2x (or whatever). It'll display as x to everyone, but even if in reality it'll be changed.

**Comment by Archer-Dante** [comment]
Sorry for not being clear enough. ^^'
I mean force everyone to see they own difficulty always, no matter if another players nearby.

Just by example:
Player 1 with X difficulty passing near/above/below/around player 2 with Y difficulty.
Any not let them get pop-up difficulty display and not let them monitor difficulty changes from this.
So player 1 won't detect player 2 somewhere around just cuz mod's feature.

Not sure about best way to implement such change.
Maybe better to make it like another option for "Area Mode"... like difficulty per player individually.
Or just like on\off option but controlled server-side (but as far as i know the whole SH isn't working like that and can't conrol client's calculations and modes and stuff without config update to the same as on server)

**Comment by Cyborgmas** [comment]
yeah, it'll definitely require some rewrite but it's manageable, I understand what you want now. However, still only going to implement it in 1.14
